jbmc/unit/java_bytecode/java_bytecode_convert_method/convert_method.cpp

```diff
@@ -308,6 +308,34 @@ class java_bytecode_convert_method_unit_testt
   {
     return converter.convert_astore(statement, op, location);
   }
+
+  static exprt variable(
+    java_bytecode_convert_methodt &converter,
+    const exprt &arg,
+    char type_char,
+    size_t address,
+    bool cast_as_needed)
```

Comment:
:pick: Why not just take `java_bytecode_convert_methodt::variable_cast_argumentt`

Reply:
this type is protected